.env.example

```diff
@@ -0,0 +1 @@
+DEBUG=
```

Comment:
Is this file used? We can drop it otherwise.

Reply:
i think the file was created wtih the python template, but i can delete it

Reply:
Delete it then; it is not needed, and it is not a problem to do something different from the template :)